pkg/services/ngalert/schedule/schedule_unit_test.go

```diff
@@ -23,7 +23,7 @@ import (
 
 	"github.com/grafana/grafana/pkg/expr"
 	"github.com/grafana/grafana/pkg/infra/tracing"
-	"github.com/grafana/grafana/pkg/plugins"
+	"github.com/grafana/grafana/pkg/plugins/manager/fakes"
```

Comment:
I wonder if the package name should be like `plugins_fakes` or something else. Otherwise there would be many `fakes` packages in different paths and in tests we will have to use aliases, and they might be different from file to file, which would not help readability.

Reply:
Yep have mentioned this in the `Special notes for your reviewer`